src/util/log.h

```diff
@@ -3,9 +3,6 @@
 
 #include <stdint.h>
 #include <stddef.h>
-#ifndef _WIN32
-#include <config.h>
-#endif
 
```

Comment:
The file [`test/integration/esys-tr-fromTpmPublic-session.int.c`](https://github.com/tpm2-software/tpm2-tss/blob/master/test/integration/esys-tr-fromTpmPublic-session.int.c) was newly created in #1442 and needs `config.h` included as well, see the [failing build](https://travis-ci.org/tpm2-software/tpm2-tss/jobs/549754600#L5776).

Non-withstanding the above, [`src/util/log.h`](https://github.com/tpm2-software/tpm2-tss/blob/master/src/util/log.h) is a bit of a special case because it relies on the definition of `MAXLOGLEVEL` in `config.h`. For this reason it might be better to keep including `config.h` here, even though it is only a header file.

Reply:
> The file test/integration/esys-tr-fromTpmPublic-session.int.c was newly created in #1442 and needs config.h included as well, see the failing build.

done

> Non-withstanding the above, src/util/log.h 

It is not different. It relays on MAXLOGLEVEL setting, it can come from config.h if this is autoconf based system which is guaranteed to be included in all files or from -DMAXLOGLEVEL= if in Windows. In any case it is guaranteed to have MAXLOGLEVEL available but not guaranteed to have config.h. We can have an alternative of having:

```
#ifndef MAXLOGLEVEL
#define MAXLOGLEVEL sane_default
#endif
```